packages/manager/apps/hpc-vmware-vsphere/src/utils/getCommercialType.ts

```diff
@@ -0,0 +1,15 @@
+import { CommercialName } from '@/types/datacenter';
+
+export type CommercialType = 'ESSENTIAL' | 'PREMIER' | 'UNKNOWN';
```

Comment:
To keep consistency with the offer type
```suggestion
export type CommercialType = 'ESSENTIALS' | 'PREMIER' | 'UNKNOWN';
```